packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -264,3 +276,35 @@ pub struct MsgIbcTransferResponse {
     /// **channel** is a src channel on neutron side trasaction was submitted from
     pub channel: String,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+/// Proposals defines the struct for various proposals which neutron may accept (currently only parameter change)
+pub struct Proposals {
```

Comment:
1. Rename to AdminProposal
2. Add a comment that currently only param change proposals are implemented, but new types of admin proposals might be implemented in the future